velox/type/Timestamp.cpp

```diff
@@ -74,17 +60,6 @@ void Timestamp::toGMT(const tz::TimeZone& zone) {
   seconds_ = sysSeconds.count();
 }
 
-void Timestamp::toGMT(int16_t tzID) {
-  if (tzID == 0) {
-    // No conversion required for time zone id 0, as it is '+00:00'.
-  } else if (tzID <= 1680) {
-    seconds_ -= getPrestoTZOffsetInSeconds(tzID);
```

Comment:
Where do we handle this branch after the change?

Reply:
this is not needed anymore. There is logic inside `tz::locateZone()` to find the right zone for offsets as well.

Reply:
so basically tz::TimeZone knows how to make this conversion now

Reply:
Got it. I see we handle it in buildTimeZoneDatabase() now. Thanks!